OMEdit/OMEditGUI/Search/SearchWidget.h

```diff
@@ -42,41 +42,68 @@
 class Label;
 class SearchFileDetails;
 class Search;
+class SearchWidgetResult;
 class SearchWidget : public QWidget
 {
   Q_OBJECT
 public:
   SearchWidget(QWidget *pParent = 0);
   ~SearchWidget();
-  QProgressBar *getProgressBar() {return mpProgressBar;}
   QComboBox *getSearchScopeComboBox() {return mpSearchScopeComboBox;}
   QComboBox *getSearchStringComboBox() {return mpSearchStringComboBox;}
   QComboBox *getSearchFilePatternComboBox() {return mpSearchFilePatternComboBox;}
+  QStackedWidget *getSearchStackedWidget() {return mpSearchStackedWidget;}
+  QComboBox * getSearchHistoryCombobox() {return mpSearchHistoryComboBox;}
   void updateComboBoxSearchStrings(QComboBox *pComboBox);
+  QList<SearchWidgetResult*> pSearchWidgetResultobjects;
```

Comment:
This is a list not a pointer. So it should be `QList<SearchWidgetResult*> mSearchWidgetResultobjects;`